UI/CardScrollUI.gd

```diff
@@ -0,0 +1,71 @@
+extends Control
+
+var card_scene: PackedScene = preload("res://Cards/Card.tscn")
+
+var discard_pile: Array[CardBase] = CardManager.card_container.discard_pile
+var draw_pile: Array[CardBase]    = CardManager.card_container.draw_pile
+var deck: Array[CardBase]         = CardManager.card_container.default_deck
+var cards_to_display: Array[CardBase] = []
+var card_worlds: Array[CardWorld] = []
+var cardworld: CardWorld = null
+
+func _on_button_pressed() -> void:
+	queue_free()
+
+#Pressing escape will return you to the game
+func _input(event) -> void:
+	if event.is_action("escape"):
```

Comment:
@Turtyo Should we add a constants file to the AutoLoader that specifies all of the different input keys we have? This would help with other screens that potentially may use "escape" input event identifier and can reference the constant itself.

Reply:
yes good idea ! It's preferable to have this kind of thing in the same place and everything using this referencing to the same place

Reply:
Could alternatively use is_key_pressed with the KEY_ESCAPE constant. I think the only buttons we’ll ever have in this game is mouse buttons and escape, so we don’t need to do anything too complicated for input probably

Reply:
Having another file where you go search your constant doesn't really complicate it (it's just referencing somewhere else basically). Not making a change because "we might not need it" is hoping that we don't make anything more complicated is generally not good (because we might indeed need it later, so it's better to have now than to have to change it later when there are more things to add). Especially when the change is simple.

We might want to add more controls later, we never know.